packages/wrangler/src/d1/execute.tsx

```diff
@@ -100,6 +109,11 @@ export const Handler = async (args: HandlerOptions): Promise<void> => {
 		logger.loggerLevel = "error";
 	}
 	await printWranglerBanner();
+	if (local) {
+		throw new UserError(
+			"`wrangler d1 execute` now defaults to local mode. Remove --local from your command to continue."
```

Comment:
Some developer experience insight here... even if the commands defaults to either local or remote, I would prefer if both `--remote` and `--local` is supported.

I'm always being explicit right now and typing `--local` and `--remote` for all of my commands because I can't honestly remember what is the default, but I'm being 'punished' by wrangler for being explicit.

Reply:
Good point.